packages/replay/test/integration/sendReplayEvent.test.ts

```diff
@@ -414,14 +411,12 @@ describe('Integration | sendReplayEvent', () => {
 
     await advanceTimers(30000);
     expect(mockSendReplayRequest).toHaveBeenCalledTimes(4);
-    expect(spySendReplay).toHaveBeenCalledTimes(4);
```

Comment:
Note: We cannot spy on recursive calls of modules, it seems, so this is never incremented. IMHO this is well enough tested by ensure that `sendReplayRequest` is called X times, we don't really care that much about `sendReplay` does.